invenio_iiif/previewer.py

```diff
@@ -27,13 +27,26 @@
 
 
 def can_preview(file):
-    """Determine if the given file can be previewed."""
+    """Determine if the given file can be previewed.
```

Comment:
If first sentence is something similar to `Determine if the given file can be previewed by its extension.`, then we don't need `This is computed according to the supported extensions.`